Libraries/Microsoft.Teams.Extensions/Microsoft.Teams.Extensions.Configuration/Microsoft.Teams.Apps.Extensions/TeamsSettings.cs

```diff
@@ -12,11 +12,15 @@ public AppOptions Apply(AppOptions? options = null)
     {
         options ??= new AppOptions();
 
-        if (ClientId is not null && ClientSecret is not null)
+        if (ClientId is not null && ClientSecret is not null && IsNotEmpty())
         {
             options.Credentials = new ClientCredentials(ClientId, ClientSecret, TenantId);
         }
 
         return options;
     }
+
+    public bool IsNotEmpty() {
```

Comment:
nit: replace this with a getter called `Empty` and use it like `!Empty`

Reply:
nice suggestion - code looks cleaner